src/components/ColorPicker/components/Slidable/Slidable.tsx

```diff
@@ -18,6 +18,25 @@ export interface Props {
   onDraggerHeight?(height: number): void;
 }
 
+let isDragging = false;
```

Comment:
Would it make more sense to add this as a private member on the class? That feels more conventional

Reply:
We need this in scope for the `window.addEventListener`, so there is no way to make this a private member of `Slidable`

Reply:
Makes sense!